services/wallet/thirdparty/opensea/client.go

```diff
@@ -35,15 +35,15 @@ const ChainIDRequiringAPIKey = 1
 
 func getbaseURL(chainID uint64) (string, error) {
 	switch chainID {
-	case 1:
+	case 1, 10, 42161:
```

Comment:
Opensea don't support arbitrum/optimism AFAIK why would u add it here?

Reply:
It does, according to the docs: https://support.opensea.io/hc/en-us/articles/4404027708051-Which-blockchains-are-compatible-with-OpenSea-

Reply:
They do list them: 
![image](https://user-images.githubusercontent.com/11161531/230398663-1624de77-1ad0-4283-b9fc-1d65989d21c8.png)

Yeap, there's no way to tell OpenSea "give me the NFTs for this specific chain" when we do a request. You'll get assets from all chains on the same response, and you can filter by looking at `"chain_identifier": "ethereum"`